components/chainhook-sdk/Cargo.toml

```diff
@@ -42,6 +42,8 @@ dashmap = "5.4.0"
 fxhash = "0.2.1"
 lazy_static = "1.4.0"
 regex = "1.9.3"
+miniscript = "10.0.0"
+bitcoin = "0.30.1"
```

Comment:
@qustavo we may not need this dep, can you confirm if we can remove?

Reply:
fixed